### ISSUE TITLE
Missing game rules

### ISSUE THREAD

**Issue by thjungers** [HereBeMike/Gridcannon#7]
Hi, 

Some rules from Tom Francis are missing (though it may have been intentional 🤷‍♂ )

- when adding armor to a royal, [Tom says ](https://www.pentadact.com/2019-08-20-gridcannon-a-single-player-game-with-regular-playing-cards/)
> the card you can’t play is added to the royal it’s most similar to (lowest value royal of same suit) and increases their health by that much

In your implementation, armor can be added to any royal.
- when no royal is alive, [Tom says](https://www.pentadact.com/2019-08-20-gridcannon-a-single-player-game-with-regular-playing-cards/)
> If there are no living royals in play: if all 12 are dead, you’ve won. If not, draw cards until you find a royal, placing these in a face-up pile as you go. Once you find a royal, place it, then add the cards you cycled through to the bottom of your deck [1]

In your implementation, you just continue drawing and placing cards.

Anyway, that's some nice work.

Cheers, 
Thomas

**Comment by HereBeMike** [comment]
Yeah, I missed some of the rules initially, I've corrected some of those, but I haven't gotten the full rules set into this version.

The armouring rule I can fix once I finish working on the new version - it's in that version.
The cycle until you find a royal to put in play rule feels weird to me, but I probably should add it to this version.
I like that you can set up the grid to take advantage of the cards you'd normally cycle through - use aces, and set traps for the next royal you draw. But sometimes it does lead to forced discards to the shame pile, which isn't nice.

Thanks for the feedback, hopefully we'll sort out these issues soon!
